src/OneExplorer/OneStorage.ts

```diff
@@ -162,4 +162,18 @@ export class OneStorage {
   public static reset(): void {
     OneStorage._obj = undefined;
   }
+
+  public static resetBaseModel(path: string): void {
+    delete OneStorage.get()._baseModelToCfgsMap[path];
+    Logger.debug('OneStorage', `Base Mode Path(${path}) is removed.`);
+  }
+
+  public static resetConfig(path: string): void {
+    delete OneStorage.get()._cfgToCfgObjMap[path];
+    Object.entries(OneStorage.get()._baseModelToCfgsMap).forEach(([modelpath]) => {
```

Comment:
I thought following code is correct but not sure... @.@
Key of `_baseModelToCfgsMap` is array?

```suggestion
    Object.entries(OneStorage.get()._baseModelToCfgsMap).forEach((modelpath) => {
```

Reply:
Object.entries returns the given object's [key, value] list in an array.
For example,
[ ['model.tflite', 'model.cfg'],
  ['A.tflite', 'A.cfg']]

 will be `Object.entries(OneStorage.get()._baseModelToCfgsMap)`

I obtained the first value as modelpath within forEach statuement by destructuring the array. So it's the key.